ListApp/ListApp.Api/App_Start/WebApiConfig.cs

```diff
@@ -11,12 +10,22 @@ public static void Register(HttpConfiguration config)
         {
             // Web API configuration and services
 
+            var constraintResolver = new DefaultInlineConstraintResolver
```

Comment:
Move resolver construction into a separate method - seems like a bit different level of abstraction than the rest of the class.